pages/common/jj-absorb.md

```diff
@@ -1,9 +1,9 @@
 # jj absorb
 
-> Split changes in the source revision and move each change to the closest mutable ancestor where the corresponding lines were modified last.
+> Split changes in the source revision and move each change to the closest mutable ancestor where the corresponding lines were modified last. Changes that have zero or multiple matching regions in ancestral revisions won't be moved.
```

Comment:
```suggestion
> Split changes in the source revision and move each change to the closest mutable ancestor where the corresponding lines were modified last.
> Changes that have zero or multiple matching regions in ancestral revisions won't be moved.
```
Better to split that for clarity

Reply:
Sounds fine to me, can you clarify repo policy on overwriting commits? Normally I'd fix this and squash the change into the commit I'm trying to push but I feel like I remember seeing conversation around that previously

Reply:
Make sure you don't force push and everything should be good. It's good to preserve PR history so you can see what changes got made after what comment. We squash mand merge the PRs so a messy PR doesn't matter.

Reply:
I usually just use the webui and click on the "commit suggestion" button. It autoresolves the conversation for convenience.